custom.proto

```diff
@@ -7,12 +7,23 @@ option cc_enable_arenas = true;
 
 package sensr_proto;
 
+message ReplayInfo {
+  int32 current_index = 1;
+}
+
+message RecordingInfo {
+  float saving_progress = 1;
+}
+
 message VehicleOrientedFieldsOfRegard {
   int32 object_id = 1;
   repeated PolygonBox field_of_regard = 2;
 }
 
 message CustomMessage {
-  repeated VehicleOrientedFieldsOfRegard vehicle_oriented_fields_of_regard = 1;
+  //repeated VehicleOrientedFieldsOfRegard vehicle_oriented_fields_of_regard = 1; // Temporary disable
+  repeated PolygonBox field_of_regard = 1;
```

Comment:
Is `field_of_regard` something old? I'm just wondering why it is assigned to the same index as `vehicle_oriented_fields_of_regard` (both are 1)

Reply:
Yes, you are right. For backward compatibility, we should use another index.
But, CustomeMessage usually used for internal purpose only. So I believe @young-seoulrobotics already updated the other side of program too.